task_processing/executors/task_executor.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import abc
+from collections import namedtuple
+
+import six
+
+TaskConfig = namedtuple(
```

Comment:
you mean coupled to specific containerization mechanism? task_config is the raw input into our runners, so we should definitely think hard about the shape of it, currently we're at the proof of concept stage.